miette-derive/src/label.rs

```diff
@@ -88,7 +99,8 @@ impl Labels {
                             span: field.span(),
                         })
                     };
-                    let LabelAttr { label } = syn::parse2::<LabelAttr>(attr.tokens.clone())?;
+                    use quote::ToTokens;
+                    let LabelAttr { label } = syn::parse2::<LabelAttr>(attr.meta.to_token_stream())?;
```

Comment:
Notable here is that I didn't see a way to get a token stream that behaved exactly the same as in syn v1, but it seemed that using `meta` while skipping the first token was suitable, so that's what I went with.